client/src/components/General/Signup/Component.jsx

```diff
@@ -116,12 +119,13 @@ class Signup extends Component {
                         <Field
                           name="confirmPassword"
                           type="password"
+                          label="Confirmation"
                           // Note: the order of validators matter
-                          validate={composeValidators(
-                            passwordRequired,
-                            passwordLength,
-                            passwordFormat
-                          )}
+                          validate={confirmation({
```

Comment:
I think we should move this validation to `client/src/utils/validationRules.js`

Reply:
Moved function to validationRules.js